README.md

```diff
@@ -202,22 +202,26 @@ pop call contract -p ./my_contract --contract $INSTANTIATED_CONTRACT_ADDRESS --m
 For end-to-end testing you will need to have a Substrate node with `pallet contracts`.
 You do not need to run it in the background since the node is started for each test independently.
 To install the latest version:
-
 ```
 cargo install contracts-node --git https://github.com/paritytech/substrate-contracts-node.git
 ```
 
-If you want to run any other node with `pallet-contracts` you need to change `CONTRACTS_NODE` environment variable:
+Run e2e testing on the Smart Contract:
 
 ```sh
-export CONTRACTS_NODE="YOUR_CONTRACTS_NODE_PATH"
+# Run e2e tests for an existing smart contract
+ pop test contract  -p ./my_contract --features e2e-tests
 ```
 
-Run e2e testing on the Smart Contract:
+If you want to run any other node with `pallet-contracts` you need to change `CONTRACTS_NODE` environment variable:
 
+```sh
+export CONTRACTS_NODE="YOUR_CONTRACTS_NODE_PATH"
+```
+You can automate this process with the `--contracts-node` flag:
```

Comment:
```suggestion
Or, use the `--contracts-node` flag:
```

This should be the first choice TMO, second doing the export

Reply:
You are right, actually the export CONTRACTS_NODE is not needed anymore